ops/services/pagerduty/_data.tf

```diff
@@ -5,7 +5,3 @@ data "pagerduty_service" "service" {
 data "pagerduty_vendor" "azure" {
   name = "Microsoft Azure"
 }
-
-data "pagerduty_vendor" "twilio" {
-  name = "Twilio"
-}
```

Comment:
does this remove our Twilio alerts or are they just configured outside of Azure?

Reply:
The Twilio alerts were configured entirely outside of terraform because Twilio doesn't support creating the necessary resources via terraform, Twilio was pointed at Pagerduty resources that already exist so this turned out to be extra and needs to be cleaned up.